src/main/java/edu/pdx/cs/sftp/User.java

```diff
@@ -4,45 +4,51 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-/** User class for SFTP project. Used to gather information about a connection */
+/** Gather information about a connection initiated by the user with the User class */
 class User {
 
-  /** Field for password string */
-  String password;
-  /** Field for username string */
+  /** Store the username in this variable */
   String username;
-  /** Field for host name string */
+  /** Store the password in this variable */
+  String password;
+  /** Store the hostname in this variable */
   String hostname;
-  /** Field for an IO scanner used for input */
+  /** Store IO scanner in this variable */
```

Comment:
I think this is a correct use of javadoc (placing it before each field), but I actually think the doc comments make the file _less_ readable in this case. Since the fields are mostly simple String types, and the variable names are clear, I don't think comments add anything here; rather, (for me) they create more of a distraction since the code is self-documenting.

My suggestion is to remove the doc comments for these fields, but I'd be interested to hear Belén's thoughts on this!

Reply:
I am good to remove them :) Agreed!

Reply:
Agreed with removing the comments on these variables. I think we briefly discussed this last class. The docstrings don't seem to be adding much more info since the variable names are pretty straightforward and descriptive.

Reply:
Exactly! Just the comments for the variable names. 👍